### ISSUE TITLE
As a user who wants to know about real candidates, we will limit typeahead to statutory candidates.

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC-web-app#785]
This is a brilliant suggestion from @boblannon that will filer out a lot of the prank filers, with beautiful names like "AAAAAAAAAA"

**Comment by PaulClark2** [comment]
I like this solution better than excluding F2Zs (re:  https://github.com/18F/openFEC/issues/964) but we'd need to explain that we are excluding certain Form 2 filers, ballot list candidates and draft committee candidates.

**Comment by LindsayYoung** [comment]
This is just for the Candidate browse view, people would default to the statutory candidates, but be able to see all candidates that filed an F2 if they unselect statutory candidate checkbox. 

We have been excluding candidates who we only have F2Z records, since those records are in poor condition. Simple data like district is wrong or missing. 

Also 18F/openFEC#964 is only affecting the cycle creation, making sure we only create a cycle dropdown for candidates that have a  F2, that is not working yet.

**Comment by LindsayYoung** [comment]
I think "C" will work it looks like there is a bug in the dev api in /candidate it looks like it is using the oldest name and status, instead of the newest. 

18F/openFEC#1244

**Comment by LindsayYoung** [comment]
Nope, the logic is right, but the receipt date on Obama's records, seems to have changed.

**Comment by noahmanger** [comment]
We've decided against this.

**Comment by boblannon** [comment]
oh, really?  I have the same problem with my typeahead, and was thinking of doing the same thing. what made you decide against? what are you doing instead?

**Comment by noahmanger** [comment]
@LindsayYoung correct me if I'm wrong, but we decided against it because of the number of legit candidates that it filtered out. We decided better to leave it off for now, but in the future add a more intelligent way of weighting results to surface the accurate ones without accidentally cutting things out. 

Also, I just realized that this issue started out referring to a statutory candidate checkbox, which we've added.
